etc/tendrl/node-agent/node-agent.conf.yaml.sample

```diff
@@ -9,7 +9,6 @@ etcd_connection: 0.0.0.0
 
 
 log_cfg_path: /etc/tendrl/node-agent/node-agent_logging.yaml
-log_level: DEBUG
 package_source_type: rpm
```

Comment:
Why remove here?

Reply:
log_level is not used anywhere,  all logging is configured using only logging.yaml.syslog.sample

Reply:
Ack